src/components/nodes-table/OTWUpdates.vue

```diff
@@ -0,0 +1,236 @@
+<template>
+	<v-container grid-list-md>
+		<v-row class="ml-5">
+			<v-col v-show="!node.firmwareUpdate" cols="12">
+				<v-row justify="center" class="mb-2 text-center" dense>
+					<v-btn
+						:disabled="loading"
+						outlined
+						class="my-auto"
+						color="green"
+						@click="checkUpdates"
+						>Check updates</v-btn
+					>
+					<v-checkbox
+						v-model="includePrereleases"
+						hide-details
+						dense
+						label="Include pre-releases"
+						class="ml-2 my-auto"
+					>
+					</v-checkbox>
+					<v-select
+						v-if="
+							controllerNode &&
+							controllerNode.RFRegion === undefined
+						"
+						style="max-width: 200px"
+						class="ml-2 mb-2"
+						label="Rf Region"
+						hide-details
+						:items="rfRegions"
+						v-model="rfRegion"
+					>
+						<template v-slot:prepend>
+							<v-icon>signal_cellular_alt</v-icon>
+						</template>
+					</v-select>
+				</v-row>
+			</v-col>
+
+			<template v-if="this.fwUpdates.length > 0 && !node.firmwareUpdate">
```

Comment:
Not sure if `node.firmwareUpdate` always has the correct state for the controller node?

Reply:
firmwareUpdate has been removed from controller in favor of dedicated events